webapp/src/app/shared/globals.ts

```diff
@@ -26,4 +27,5 @@ export class Globals {
 	static phaserEventsService: PhaserEventsService;
 	static autotileService: AutotileService;
 	static entityRegistry: EntityRegistryService;
+	static modloaderService: ModloaderService;
```

Comment:
Do not add more services to global if possible. As the `//TODO` suggests we are trying to remove them.